packages/ketcher-react/src/script/ui/views/components/ContextMenu/ContextMenu.tsx

```diff
@@ -0,0 +1,182 @@
+/****************************************************************************
```

Comment:
I propose to move this file to the same folder, where existing context menu is located (or vice versa):
<img width="304" alt="image" src="https://user-images.githubusercontent.com/41272128/206489785-5dc16410-fedb-4de3-b8c5-65ad59a12b69.png">

Reply:
You are right I've noticed that. I recommend putting them in the folder where `StructEditor` is, because only `StructEditor` needs them. And I think it will be better to do this when refactoring the functional group context menu.